components/dashboard/src/onboarding/StepOrgInfo.tsx

```diff
@@ -0,0 +1,180 @@
+    const updateUser = useUpdateCurrentUserMutation();
+    const jobRoleOptions = useMemo(getJobRoleOptions, []);
+    const signupGoalsOptions = useMemo(getSignupGoalsOptions, []);
+
+    const [jobRole, setJobRole] = useState(user.additionalData?.profile?.jobRole ?? "");
+    const [jobRoleOther, setJobRoleOther] = useState(user.additionalData?.profile?.jobRoleOther ?? "");
+    const [signupGoals, setSignupGoals] = useState<string[]>(user.additionalData?.profile?.signupGoals ?? []);
+    const [signupGoalsOther, setSignupGoalsOther] = useState(user.additionalData?.profile?.signupGoalsOther ?? "");
+    const [companyWebsite, setCompanyWebsite] = useState(user.additionalData?.profile?.companyWebsite ?? "");
+
+    const addSignupGoal = useCallback(
+        (goal: string) => {
+            if (!signupGoals.includes(goal)) {
+                setSignupGoals([...signupGoals, goal]);
+            }
+        },
+        [signupGoals],
+    );
+
+    const removeSignupGoal = useCallback(
+        (goal: string) => {
+            if (signupGoals.includes(goal)) {
+                const idx = signupGoals.indexOf(goal);
+                const newGoals = [...signupGoals];
+                newGoals.splice(idx, 1);
+                setSignupGoals(newGoals);
+            }
+            // clear out freeform other if removing option
+            if (goal === SIGNUP_GOALS_OTHER) {
+                setSignupGoalsOther("");
+            }
+        },
+        [signupGoals],
+    );
+
+    const handleSubmit = useCallback(async () => {
+        const additionalData = user.additionalData || {};
+        const profile = additionalData.profile || {};
+
+        const updates = {
+            additionalData: {
+                ...additionalData,
+                profile: {
+                    ...profile,
+                    jobRole,
+                    jobRoleOther,
+                    signupGoals: signupGoals.filter(Boolean),
+                    signupGoalsOther,
+                    companyWebsite,
+                },
+            },
+        };
+
+        try {
+            const updatedUser = await updateUser.mutateAsync(updates);
+            onComplete(updatedUser);
+        } catch (e) {
+            console.error(e);
+        }
+    }, [
+        companyWebsite,
+        jobRole,
+        jobRoleOther,
+        onComplete,
+        signupGoals,
+        signupGoalsOther,
+        updateUser,
+        user.additionalData,
+    ]);
+
+    const jobRoleError = useOnBlurError("Please select one", !!jobRole);
+    const isValid = jobRoleError.isValid && signupGoals.length > 0;
+
+    return (
+        <OnboardingStep
+            title="Tell us more about you"
+            subtitle="Let us know what brought you here."
+            error={updateUser.isError ? "There was a problem saving your answers" : ""}
+            isValid={isValid}
+            isLoading={updateUser.isLoading}
+            onSubmit={handleSubmit}
+        >
```

Comment:
thought (ux): Can we get something like a <kbd>Skip</kbd> button here? Maybe also in the step before

Reply:
Agree, and also

- `Skip` to skip whole onboarding flow
- Another `Skip` to skip current step
- Maybe we can also provide a way to show how many steps we have, and where we are now
- Change the last step `Continue` button title to `Complete`?